.github/workflows/package-js-api.yml

```diff
@@ -0,0 +1,20 @@
+name: Tests for woocommerce/api JS package
+
+on:
+  pull_request:
+    paths:
+      - 'packages/js/api/**'
+      - '!**.md'
```

Comment:
This works on my end as well. I tested it on a [dummy PR](https://github.com/woocommerce/woocommerce/runs/5354559772?check_suite_focus=true). This workflow wasn't triggered when I edited an MD file. But it was triggered correctly when I edited a non-MD one. Looking good to me.